uint8 message
uint8 valeur1
uint8 valeur2
